root/etc/e-smith/templates/opt/janus/etc/janus/janus.cfg/10base

```
@@ -90,7 +98,15 @@ nice_debug = false
 ; such, use the option with caution and only if you know what you're doing.
 ; Besides, it's still recommended to also enable STUN in those cases,
 ; and keep ICE Lite disabled as it's not strictly speaking a public server.
-;nat_1_1_mapping = 1.2.3.4
+
+{
+    my $mode = ${'janus-gateway'}{natMode} || 'stun';
+    if ($mode eq '1:1') {
+        my $ip = ${'janus-gateway'}{publicIP} || die ('publicIP missing');
```

Try to not fail hard here.

I'd change the line 104 with something like:
```
if ($mode eq '1:1' && $publicIp ne '') {
```